blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,138 @@
+
+   For example, below command can be used to enable Prometheus metrics visualizing for `testuser-0` user.
+
+   `oc adm policy add-cluster-role-to-user cluster-monitoring-view  testuser-0`
+
+## Network Observability metrics use case:
+Let's look at a scenario how users can benefit from metrics published by Network Observability Operator. For instance, if you suspect anomaly with DNS lookups in your cluster and want to investigate workloads that may be facing DNS latencies. With Network Observability's `DNSTracking` feature and enriched Prometheus metrics you can quickly set up an alert to trigger on high DNS latencies.
+
+For example, below alert will trigger for any workloads that experiences DNS latency > 100ms: 
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: dns-latency-alert
+  namespace: netobserv
+spec:
+  groups:
+  - name: DNSLatencyAlert
+    rules:
+    - alert: DNSLatencyAlert
+      annotations:
+        message: |-
+          {{ $labels.DstK8S_OwnerName }} in {{ $labels.DstK8S_Namespace }} is experiencing high DNS Latencies.
+        summary: "Trigger for any workloads experiencing > than 100ms DNS Latency."
+      expr: histogram_quantile(0.9, sum(rate(netobserv_workload_dns_latency_seconds_bucket{DstK8S_Namespace!=""}[2m])) by (le,DstK8S_Namespace,DstK8S_OwnerName))*1000 > 100
+      for: 10s
+      labels:
+        severity: warning
+```
+
+To demonstrate this use-case, I configured CoreDNS's [erratic plugin](https://coredns.io/plugins/erratic/) in openshift-dns to add latencies for `example.org` domain using below config:
+
+```
+example.org {
+        erratic {
+            delay 2 100ms
+        }
+}
+```
+
+Above config add 100ms delay to every 2nd DNS request coming in for example.org. A test pod performing DNS lookups for `example.org` every 1 second was created, eventually triggering earlier configured `DNSLatencyAlert` in my OCP cluster.
+
+![DNSLatency alert triggered for threshold > 100ms](images/dns_latency_alert_firing.png)
+
+Similarly, additional alerts on different DNS response codes could be set up, for example an alert for DNS lookup failures such as DNS queries receiving NXDOMAIN or SERVFAIL responses can also be set up as flowlogs and metrics are already enriched with DNS response codes.
+
+In addition to metrics for `DNSTracking` feature, Network Observability provides metrics for other features such as Round-Trip-Times and Packet Drops as well.
+
+## Conclusion and next steps:
+
+Network Observability operator provides the visibility you need to proactively detect issues within OpenShift cluster networking. Now with an option to disable loki, Network Observability operator provides light weight solution to visualize, diagnose and troubleshoot networking issues faster at a lower cost. Network Observability's Prometheus metrics can be leveraged to set up user defined alerts in your OCP cluster.
```

Comment:
> to set up user defined alerts in your OCP cluster

about this text, does this sound like a compulsion that user has to create alerts? I want to make it sound like on the lines of "and instead leverage Prometheus metrics which can be optionally used to set up user defined alerts in your OCP cluster", wdyt? or the sentence is getting too complex if we add that part?

Reply:
You're right. The "to set up" is definitely seems more like the customer needs to do it. I wonder if the wording more along the lines of "the ability" to do these things would be more appropriate? I hesitate to use "option" again since we just use it in the "Now with an option to disable Loki". Maybe something like this:

> Now with an option to disable Loki, and the ability to leverage Prometheus metrics to set up user defined alerts in your OCP cluster, the Network Observability Operator provides a light-weight solution to visualize, diagnose and troubleshoot networking issues faster and at a lower cost.

Reply:
how about we just leave out the alerts part in the conclusion, it's one of the use cases of how users can leverage these metrics, but not a something we're providing:
```suggestion
The Network Observability Operator provides the visibility you need to proactively detect issues within OpenShift cluster networking. Now with an option to disable Loki, and instead leverage Prometheus metrics, the Network Observability Operator provides a light-weight solution to visualize, diagnose and troubleshoot networking issues faster and at a lower cost. 
```